[2022-12-03 15:03:16,720] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: prepare_data.get_last_id scheduled__2022-11-28T19:09:00+00:00 [queued]>
[2022-12-03 15:03:17,682] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: prepare_data.get_last_id scheduled__2022-11-28T19:09:00+00:00 [queued]>
[2022-12-03 15:03:17,683] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-12-03 15:03:17,683] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-12-03 15:03:17,683] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-12-03 15:03:19,058] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): get_last_id> on 2022-11-28 19:09:00+00:00
[2022-12-03 15:03:19,288] {standard_task_runner.py:52} INFO - Started process 965 to run task
[2022-12-03 15:03:19,543] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'prepare_data', 'get_last_id', 'scheduled__2022-11-28T19:09:00+00:00', '--job-id', '277', '--raw', '--subdir', 'DAGS_FOLDER/prepare_data.py', '--cfg-path', '/tmp/tmp4t5oc28l', '--error-file', '/tmp/tmpfmpzkqm8']
[2022-12-03 15:03:19,683] {standard_task_runner.py:80} INFO - Job 277: Subtask get_last_id
[2022-12-03 15:03:22,328] {task_command.py:369} INFO - Running <TaskInstance: prepare_data.get_last_id scheduled__2022-11-28T19:09:00+00:00 [running]> on host b41792e8e843
[2022-12-03 15:03:26,071] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=prepare_data
AIRFLOW_CTX_TASK_ID=get_last_id
AIRFLOW_CTX_EXECUTION_DATE=2022-11-28T19:09:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-11-28T19:09:00+00:00
[2022-12-03 15:03:26,720] {base.py:68} INFO - Using connection ID 'postgres_default' for task execution.
[2022-12-03 15:03:27,865] {python.py:173} INFO - Done. Returned value was: None
[2022-12-03 15:03:30,438] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=prepare_data, task_id=get_last_id, execution_date=20221128T190900, start_date=20221203T150316, end_date=20221203T150330
[2022-12-03 15:03:33,346] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-12-03 15:03:35,485] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
